apps/web/modules/getting-started/[[...step]]/onboarding-view.tsx

```diff
@@ -85,13 +84,14 @@ const stepRouteSchema = z.object({
 
 type PageProps = {
   hasPendingInvites: boolean;
+  user: RouterOutputs["viewer"]["me"]["get"];
 };
+
 const OnboardingPage = (props: PageProps) => {
   const pathname = usePathname();
   const params = useParamsWithFallback();
-
+  const user = props.user;
   const router = useRouter();
-  const [user] = trpc.viewer.me.get.useSuspenseQuery();
```

Comment:
For some reason, the trpc context was recognizing a logged-in user as `unauthorized` in the first attempt of the trpc call in the first mount. Not sure why this is only happening in `/getting-started` page, I could dig deeper, but I have other stuff as well, so I stopped investigating further and did a fix by doing server-side fetching for the user info. @emrysal